cmd/oceanbench/monitor.go

```
@@ -149,32 +149,60 @@ func monitorHandler(w http.ResponseWriter, r *http.Request) {
 	writeTemplate(w, r, "monitor.html", &data, "")
 }
 
+func scalarCount(ctx context.Context, device model.Device, start, end int64) (count int, err error) {
+	for _, pin := range strings.Split(device.Inputs, ",") {
+		if count != 0 || (pin[0] != 'A' && pin[0] != 'D' && pin[0] != 'X') {
+			continue
```

I'm curious if there is ever a time where count is != 0 that we wouldn't want to return an error, or at very least break?
from my perspective, it either seems that a programmer error has occurred, or that we are going to loop over every pin unnecessarily?

---

Yeah I kinda agree, but not super interested in changing how this works in this PR.

---

Yeah seems reasonable, I was kinda just curious honestly